### DISCUSSION TITLE
Where did my old posts go?

### DISCUSSION THREAD

**Discussion by gitcos** [community/community#35815] [Discussions] [answered]
Earlier this year I made a few posts seeking help with different aspects of GitHub Actions.  Although I didn't get solutions from those posts, I did save the links to refer to later, for complete descriptions of the issues I faced.  Now, I can't find any of those posts!!

All my saved links redirect to the new main page for discussions.  Searching for the titles does not find the original posts.

Here are some of my saved links:

  https://github.community/t/how-to-conditionally-set-a-jobs-job-id-foo-value/238765
  https://github.community/t/how-to-get-the-list-of-files-that-triggered-push-event/17982
  https://github.community/t/path-filter-seems-to-always-fail/238783

Does anyone know where to find these posts?  All of them are from this year, I believe.

**Comment by airtower-luna** [comment, marked as answer]
If they didn't have a marked answer, I'm afraid they are gone. A staff member stated here that there was a decision to migrate only threads with solutions: https://github.com/orgs/community/discussions/27346#discussioncomment-3275305 :confused:

**Reply by gitcos** [reply]
Ugh!  That's really really frustrating.  Is there any way to get them back?

**Reply by airtower-luna** [reply]
Not that I'm aware of. I've seen other threads with similar questions, but no restoration. :slightly_frowning_face:

**Comment by gitcos** [comment]
Is the person who explained that, jricciardi, even on this forum?  I tried tagging @jricciardi and no matches came up.

**Comment by gitcos** [comment]
@ettaboyle also does not seem to be here.  Did they abandon the forum after the migration?
